ci/base-images/amdgpu-cray-mpich/HOWTO.md

```diff
@@ -0,0 +1,43 @@
+# Modified base image to allow building cray-mpich
+
+## Preparation steps
+
+```
+mkdir lib64
+cp -a /usr/lib64/libxpmem.* lib64/
+
+git clone https://github.com/eth-cscs/alps-cluster-config.git
+
+cp alps-cluster-config/beverin/packages.yaml packages.yaml
+```
+
+## Edit cluster config files
+
+Modify `packages.yaml`:
+```
+   patchelf:
+     require: "@:0.17"
+   libfabric:
+-    buildable: false
+-    externals:
+-    - spec: libfabric@1.15.2.0
+-      prefix: /opt/cray/libfabric/1.15.2.0/
++    require: "@1.15.2"
```

Comment:
Should we be building with 1.22.X now that daint has been updated? I think that's the version the container engine will inject nowadays. I also think building with 1.15 will keep working as they are ABI compatible. I also don't think we're affected by the issues that NCCL/aws-ofi-nccl has with different libfabric versions, but not 100% sure.